### PR TITLE
added crucial documentation on SELU activation

### PR DESCRIPTION
**Pull request by gklambauer**
Users extrememly frequently forget to use the correct initialization with SELUs. Added notes to documentation to use correct initialization wherever possible.

### CODE REVIEW

**Comment by KingRando95** [comment]
Get Outlook for Android<https://aka.ms/ghei36>



From: G K
Sent: M12 13, Wed 3:49 AM
Subject: [tensorflow/tensorflow] added crucial documentation on SELU activation (#15337)
To: tensorflow/tensorflow
Cc: Subscribed


Users extrememly frequently forget to use the correct initialization with SELUs. Added notes to documentation to use correct initialization wherever possible.
You can view, comment on, or merge this pull request online at:
  https://github.com/tensorflow/tensorflow/pull/15337
Commit Summary
added crucial documentation on SELU activation
File Changes
M tensorflow/core/api_def/base_api/api_def_Selu.pbtxt<https://github.com/tensorflow/tensorflow/pull/15337/files#diff-0> (5) M tensorflow/go/op/wrappers.go<https://github.com/tensorflow/tensorflow/pull/15337/files#diff-1> (2) M tensorflow/python/keras/_impl/keras/activations.py<https://github.com/tensorflow/tensorflow/pull/15337/files#diff-2> (4)
Patch Links:
https://github.com/tensorflow/tensorflow/pull/15337.patch https://github.com/tensorflow/tensorflow/pull/15337.diff

**Comment by gklambauer** [comment]
I signed it!

**Comment by gklambauer** [comment]
Do I have to do anything?

**Comment by drpngx** [comment]
No, just hang tight until @fchollet gets around to it.

**Review by fchollet — COMMENTED** [review]
Thank you for your patience.

File: tensorflow/core/api_def/base_api/api_def_Selu.pbtxt [hunk 1]
```diff
@@ -4,6 +4,11 @@ op {
   description: <<END
 if < 0, `scale * features` otherwise.
 
+To be used together with:
+`initializer = layers.variance_scaling_initializer(factor=1.0, mode='FAN_IN')`
+and
+'tf.contrib.nn.alpha_dropout'
```
[the remark this answers is not in the document]
Comment (gklambauer) [reply tensorflow/core/api_def/base_api/api_def_Selu.pbtxt]: Just change accordingly... I am not familiar with policies concerning "contrib" and stuff.

**Comment by raghuraman-k** [comment]
@fchollet: Does this look good to you?

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @fchollet: It has been 41 days with no activity and the `awaiting review` label was assigned. Can you please take a look?

**Review by fchollet — DISMISSED** [review]
LGTM, sorry for the delayed review.

**Comment by drpngx** [comment]
@gklambauer could you pull rebase and push again?

**Review by fchollet — DISMISSED** [review]

**Comment by fchollet** [comment]
Sorry for the delay in getting this tested. Could you please rebase to fix the merge conflicts?

**Review by drpngx — APPROVED** [review]
Approve the last two commits on top of @fchollet last review.
